pkgs/os-specific/linux/decklink/default.nix

```diff
@@ -0,0 +1,59 @@
+{ stdenv, requireFile, fetchpatch, kernel }:
+
+stdenv.mkDerivation rec {
+  pname = "decklink";
+  version = "11.6";
+
+  src = requireFile {
+    name = "Blackmagic_Desktop_Video_Linux_11.6.tar.gz";
+    url = "https://www.blackmagicdesign.com/support/download/d399ada95c2b49ffad3031bda413acb5/Linux";
+    sha256 = "0qwm1b3gy0k7j1bimkxwwr77g8hrsybs9jp90b46kzcy06mcp380";
+  };
+
+  patches = [
+    (fetchpatch {
+      url = "https://aur.archlinux.org/cgit/aur.git/plain/01-fix-makefile.patch?h=decklink";
```

Comment:
Please add a revision to these urls so that it doesn't break when the patch is changed.